monitor.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 mem=$(free -m | awk 'NR==2{printf("%d\n", $3*100/$2)}')
-disk=$(df -h | awk '$NF=="/"{print $5}' | awk -F% '{print $1}')
+disk=$(df -h | awk '$NF=="/" {sub(/%/, "", $5); print $5}')
```

Comment:
The `awk` function `sub()` allows for substitutions, or global if you use `gsub()`.